source/new.rst

```diff
@@ -479,38 +483,36 @@ We use this message exchange
 to also ask Bob wether he agrees to becoming part of the group.
 
 The protocol re-uses the first five steps of the `setup-contact`_ protocol
-(with small modifications)
 so that Alice and Bob verify each other's keys.
-We make small modifications to indicate that
-the messages are part of the verified group protocol,
-to include the group's identifier,
-and to ask for Bob's explicit consent.
-More precisely:
+To ask for Bob's explicit consent we
+indicate that the messages are part of the verified group protocol,
+and include the group's identifier
+in the metadata transfered in the out of band channel.
 
-- we substitute the message prefix "vc-" by "vg-".
+More precisely:
 
-- in step 1 there are two changes.
-  We change the type of the out-of-band transferred to ``TYPE=vg-invite``.
-  Second, Alice adds the name of the group ``GROUP`` to the bootstrap code
-  to indicate that Alice offers Bob to join the group ``GROUP``.
+- in step 1 Alice adds the metadata:
+  ``INVITE=[groupname]``.
+  Where ``[groupname]`` is the name of the group ``GROUP``.
```

Comment:
any particular reason to use ``[...]`` as a placeholder here? I'd rather go for ``<...>`` or ``$GROUPNAME`` to borrow from shell-land.  The parentheses are a bit overloaded especially with e-mail.